app/src/config/constants.ts

```diff
@@ -13,3 +13,7 @@ export const ADOBE_EMBED_URL =
 export const TRUNCATED_LENGTH = 80;
 
 export const CARDS_PER_PAGE = 48;
+
+export const DEFAULT_PAGE_NUM = 1;
```

Comment:
Not necessary now, but as this app adds more pages and it will have multiple searches, this might be tricky to maintain. For example, you might have `DEFAULT_COLLECTION_SORT`, `DEFAULT_MAIN_SEARCH_SORT`, etc for any page also need as sort. 

Perhaps `DEFAULT_COLLECTION_SORT` can have a more generic name, or there's a file for `/collection` page constant and utils. Again, not necessary now but keep that in mind.